src/onboarding/Start.js

```diff
@@ -74,7 +76,13 @@ class Start extends React.Component {
     return (
       <Main style={{ opacity: screenTransitionStyles.opacity }}>
         <Content style={screenTransitionStyles}>
-          <StartContent
+          <BreakPoint to="medium">
+            <Warning>
+              DAO creation is desktop only for now, please use aragon.org on
+              desktop for it
```

Comment:
This sentence reads a bit clumsy to me? 🧐 Also we should always use “Organization” rather than “DAO” in the UI, and the period is missing.